harvesting/oaipmh.py

```diff
@@ -46,7 +46,7 @@ def list_records(dois):
             continue
 
         try:
-            redirect_url = _get_redirect(software)
+            redirect_url = _get_redirect(software, headers)
```

Comment:
good find

Reply:
On second thought, `headers` contains authorization information meant for Zenodo, right? But here, we're giving that information to doi.org. Not sure if it is used/passed on there.

Reply:
That is correct. My suggested solution sends Zenodo headers to doi.org, which makes no sense. Rather than adding a headers parameter to `_get_redirect`, we should remove the `headers=headers` parameter from the `requests.head` call to doi.org on line 108.

Reply:
FYI Branched https://github.com/research-software-directory/research-software-directory/pull/489 off of this PR.